specification/properties/Address.json

```diff
@@ -25,7 +25,7 @@
   },
   "region_code": {
     "type": "string",
-    "pattern": "[A-Z]{3}",
+    "pattern": "^$|^[A-Z]{2,3}$",
```

Comment:
I don't think we should allow empty strings, it should just be omitted if not set.